### DISCUSSION TITLE
Switching between pages in Dark mode results in white page in between

### DISCUSSION THREAD

**Discussion by mackerel225** [requarks/wiki#7385] [Help / Questions] [answered]
When you navigate between pages in dark mode, a white page appears momentarily for a split second before you are redirected making the experience unpleasant. Is there a way to disable this? Or change the white page to dark so it doesn't hurt eyes

**Comment by NGPixel** [comment, marked as answer]
https://feedback.js.wiki/wiki/p/avoid-complete-site-reload-when-navigating
